README.md

```diff
@@ -136,40 +136,48 @@ directly with `poetry` rather than go through the package.
 brew install mysql postgresql # MacOS dependencies for C bindings
 poetry install
 ```
-
-**2. Download CSV of Testing Data**
+**2. Start Databases**
 
 ```shell-session
-wget https://files.grouplens.org/datasets/movielens/ml-25m.zip
-unzip ml-25m.zip -d dev/
+docker-compose up -d mysql postgres
 ```
 
-**3. Start Databases**
+**3. Run Unit Tests**
 
 ```shell-session
-docker-compose up -d mysql postgres
+poetry run python3 -m unittest
 ```
 
-**4. Run Unit Tests**
+**4. Seed the Database(s)**
+
+First, download the CSVs of seeding data:
 
 ```shell-session
-poetry run python3 -m unittest
-```
+wget https://files.grouplens.org/datasets/movielens/ml-25m.zip
+unzip ml-25m.zip -d dev/
+
+# By default, we insert 25M rows which takes manu minutes
+# Optional, but recommended, only 1M rows for speed
+# Seedings always happens from `ratings.csv`, get 1M rows (+header)
+mv dev/ml-25m/ratings.csv dev/ml-25m/ratings_25m.csv
+head -n 1000001 dev/ml-25m/ratings_25m.csv > dev/ml-25m/ratings.csv
 
-**5. Seed the Database(s)**
+```
 
-If you're just testing, we recommend just setting up one database (e.g.
-Postgres) to avoid incurring the long setup time repeatedly.
+Now you can insert it into the testing database(s):
 
 ```shell-session
 preql -f dev/prepare_db.pql postgres://postgres:Password1@127.0.0.1:5432/postgres
+# You can compare in just one if you like, you don't need to seed them all.
```

Comment:
This comment here is out of place. But we can write at the top "These are optional, based on what dbs you have. Just one of them is enough to run everything."

Also they *need* mysql to run tests. Unless we switch the default to postgres.

Reply:
Ya, but you don't _need_ to populate MySQL to run tests (and we should keep it that way)

Reply:
Fair point